medical_insurance/models/medical_insurance_template.py

```diff
@@ -26,10 +9,6 @@ class MedicalInsuranceTemplate(models.Model):
     _name = 'medical.insurance.template'
     _description = 'Medical Insurance Templates'
     _inherits = {'product.product': 'product_id', }
-    name = fields.Char(
```

Comment:
If we remove name, we should probably add a function `name_get` or at least set a `_rec_name`

Maybe 

```
    @api.multi
    @api.depends('product_id.name',
                 'insurance_company_id.name',
                 'plan_number')
    def name_get(self):
        result = []
        for r in self:
            name = "%s (%s,%s)" % (
                product_id.name,
                insurance_company_id.name,
                plan_number
            )
            result.append((r.id, name))
        return result
```